packages/@react-stately/select/src/useSelectState.ts

```diff
@@ -65,7 +76,8 @@ export function useSelectState<T extends object>(props: SelectStateOptions<T>):
     },
     toggle(focusStrategy) {
       if (listState.collection.size !== 0) {
-        triggerState.toggle(focusStrategy);
+        setFocusStrategy(focusStrategy);
```

Comment:
`useMenuTrigger` also normalized `undefined` and `null`. do we need to do that too?